docs/src/domains/cdn/cloudflare.md

```diff
@@ -1,69 +1,71 @@
-          https                       https
-User <---------------> Cloudflare <-------------> Platform.sh
-```
+An on-path attack occurs when a hacker intercepts or modifies the communication between a client and a server.
+This can lead to sensitive data leaks.
+To prevent such attacks, make sure all communication with your site is encrypted through HTTPS
+and can't be downgraded to HTTP.
 
-The other option known as **Flexible SSL** causes issues if you intend to redirect all traffic to HTTPS.
-The **Flexible SSL** option uses Cloudflare's TLS/SSL certificate to encrypt traffic between your users and the CDN,
-but passes requests from the CDN back to your project at Platform.sh via HTTP.
-This facilitates sites that don't have a TLS/SSL certificate beginning to offer their users a more secure experience,
-by at the least eliminating the unencrypted attack vector on the "last mile" to the user's browser.
+To do so, [enable full strict SSL/TLS encryption](https://developers.cloudflare.com/ssl/origin-configuration/ssl-modes/full-strict/).
+Any communication between a client and Cloudflare 
+or between Cloudflare and your Platform.sh server is then encrypted through HTTPS.
+In addition, Cloudflare checks that your Platform.sh server's [TLS certificate](../../other/glossary.md#transport-layer-security-tls) 
+was issued by a trusted certificate authority.
+This confirms the client is truly communicating with your Platform.sh server.
 
-```text
-# Cloudflare's Flexible SSL option
-          https                       http
-User <---------------> Cloudflare <-------------> Platform.sh
-```
+For enhanced security, make sure your HTTPS connections can't be downgraded to HTTP.
+To do so, in your Cloudflare account,
+[enable HTTP strict transport security (HSTS)](https://developers.cloudflare.com/ssl/edge-certificates/additional-options/http-strict-transport-security/).
 
-This causes all traffic from Cloudflare to your project to be redirected to HTTPS,
-which sets off an endless loop as HTTPS traffic is presented as HTTP to your project no matter what.
+## 5. Let's Encrypt users: Renew your TLS certificates
 
-In short: *Always use **Full SSL** unless you have a very clear reason to do otherwise*
+To secure connections and prevent data leaks,
+authenticate your Platform.sh server with a [transport layer security (TL
+S) certificate](../../other/glossary.md#transport-layer-security-tls).
 
-## Let's Encrypt certificate renewal
+TLS certificates are provided by trusted certificate authorities (CA) such as Let's Encrypt.
+If you use TLS certificates provided by Let's Encrypt,
+you need to renew them [every 90 days](https://letsencrypt.org/docs/faq/#what-is-the-lifetime-for-let-s-encrypt-certificates-for-how-long-are-they-valid). 
 
-Let's Encrypt expects the `.well-known` endpoint on all domains added.
-You have 2 options:
+To ensure such renewals succeed, remove all domains pointing to Cloudflare from your Platform.sh project 
```

Comment:
:question: The `domains pointing to Cloudflare` part confused me a little because I started thinking that there was something in the Platform.sh project that was pointing to Cloudflare. That isn't true, though, right? I guess I just don't understand why this part is recommended, especially when the renewal is automated.

Reply:
Yes, I was [suspicious about this section](https://github.com/platformsh/platformsh-docs/pull/2746#discussion_r1047386499) from the get go, and your comments only add fuel to the idea that it should be removed.